From: Kim <kimw@users.noreply.github.com>
Date: Tue, 20 Mar 2018 13:54:49 +0800
Subject: Update plugin.c

Launch plugin with NULL params gives plugin to run in both plugin mode & cli mode.

See https://github.com/cbeuw/GoQuiet/issues/15
(cherry picked from commit d61b8f0c338788b47f3cbc014296c27669178d73)
---
 src/plugin.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/plugin.c b/src/plugin.c
index c3d2f5a..cf7b9b4 100644
--- a/src/plugin.c
+++ b/src/plugin.c
@@ -96,7 +96,7 @@ start_ss_plugin(const char *plugin,
     if (plugin_opts != NULL)
         cork_env_add(env, "SS_PLUGIN_OPTIONS", plugin_opts);
 
-    exec = cork_exec_new(plugin);
+    exec = cork_exec_new_with_params(plugin, NULL);
 
     cork_exec_set_env(exec, env);
 
